src/OrbitGl/App.cpp

```diff
@@ -79,6 +79,7 @@
 #include "MetricsUploader/orbit_log_event.pb.h"
 #include "ObjectUtils/Address.h"
 #include "ObjectUtils/ElfFile.h"
+#include "OrbitBase/CanceledOr.h"
```

Comment:
Nit: `CancelledOr`

Reply:
I thought about this for a longer time than I want to admit 😅
I thought also canceled looks a bit weird, but apparently this spelling is more common in american english. "Cancelled" is the british english spelling